config/standard/translations/messages-sw.properties

```diff
@@ -0,0 +1,95 @@
+
+targets.active_pregnancies.title = 
+targets.all_time.subtitle = 
+targets.pregnancy_registrations.title = 
+targets.births.title = 
+targets.delivery_1_visit.title = 
+targets.delivery_4_visits.title = 
+targets.facility_deliveries.title = 
+targets.active_pnc.title = 
+targets.pnc_registrations.title = 
+targets.pnc_visits.title = 
+targets.homebirth_no_visits.title = 
+targets.homebirth_1_visit.title = 
+targets.birth_3_visits.title = 
+targets.children_u5.title = 
+targets.children_registered.title = 
+targets.vaccines_given.title = 
+targets.children_vaccinated.title = 
+targets.past_3mos.subtitle = 
+targets.children_not_vaccinated.title = 
+targets.bcg_reported.title = 
+targets.this_month.subtitle = 
+
+schedule.child_health = Child Health: Group {{group}}
+messages.schedule.child.month_01 = 
+messages.schedule.child.month_02 = 
+messages.schedule.child.month_03 = 
+messages.schedule.child.month_04 = 
+messages.schedule.child.month_05 = 
+messages.schedule.child.month_06 = 
+messages.schedule.child.month_07 = 
+messages.schedule.child.month_08 = 
+messages.schedule.child.month_09 = 
+messages.schedule.child.month_10 = 
+messages.schedule.child.month_11 = 
+messages.schedule.child.month_12 = 
+messages.schedule.child.month_13 = 
+messages.schedule.child.month_14 = 
+messages.schedule.child.month_15 = 
+messages.schedule.child.month_16 = 
+messages.schedule.child.month_17 = 
+messages.schedule.child.month_18 = 
+messages.schedule.child.month_19 = 
+messages.schedule.child.month_20 = 
+messages.schedule.child.month_21 = 
+messages.schedule.child.month_22 = 
+messages.schedule.child.month_23 = 
+messages.schedule.child.month_24 = 
+messages.schedule.child.month_25 = 
+messages.schedule.child.month_26 = 
+messages.schedule.child.month_27 = 
+messages.schedule.child.month_28 = 
+messages.schedule.child.month_29 = 
+messages.schedule.child.month_30 = 
+messages.schedule.child.month_31 = 
+messages.schedule.child.month_32 = 
+messages.schedule.child.month_33 = 
+messages.schedule.child.month_34 = 
+messages.schedule.child.month_35 = 
+messages.schedule.child.month_36 = 
+messages.schedule.child.month_37 = 
+messages.schedule.child.month_38 = 
+messages.schedule.child.month_39 = 
+messages.schedule.child.month_40 = 
+messages.schedule.child.month_41 = 
+messages.schedule.child.month_42 = 
+messages.schedule.child.month_43 = 
+messages.schedule.child.month_44 = 
+messages.schedule.child.month_45 = 
+messages.schedule.child.month_46 = 
+messages.schedule.child.month_47 = 
+messages.schedule.child.month_48 = 
+messages.schedule.child.month_49 = 
+messages.schedule.child.month_50 = 
+messages.schedule.child.month_51 = 
+messages.schedule.child.month_52 = 
+messages.schedule.child.month_53 = 
+messages.schedule.child.month_54 = 
+messages.schedule.child.month_55 = 
+messages.schedule.child.month_56 = 
+messages.schedule.child.month_57 = 
+messages.schedule.child.month_58 = 
+messages.schedule.child.month_59 = 
+messages.schedule.child.month_60 = 

```

Comment:
We should just merge all the standard translations into the app translations in `ddocs/_attachments/translations` - a lot of them are there anyway.